src/components/Tabs.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import './Tabs.scss';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const hangleClick = (event, id) => {
+    event.preventDefault();
+    onTabSelected(id);
+  };
+
+  return (
+    <ul className="tab">
+      {tabs.map(tab => (
+        <li key={tab.id}>
+          <button
+            type="button"
+            className={classnames({
+              tab__item: true,
+              active: tab.id === selectedTabId,
+            })}
+            onClick={event => hangleClick(event, tab.id)}
```

Comment:
just use onTabSelected, remove handleClick

```suggestion
            onClick={event => onTabSelected(tab.id)}
```